Source/Editor/DualityEditor/DualityEditorApp.cs

```diff
@@ -63,6 +63,8 @@ public static class DualityEditorApp
 		private	static AutosaveFrequency			autosaveFrequency	= AutosaveFrequency.ThirtyMinutes;
 		private	static DateTime						autosaveLast		= DateTime.Now;
 		private	static string						launcherApp			= null;
+		private static string                       lastOpenScene       = null;
```

Comment:
Suggestion: The `lastOpenScene` could actually be a `ContentRef<Scene>`, which would make the fields usage a bit clearer, and also simplify the code you're using to switch if it exists.

Would merge either way, but putting this here in case you agree that this would be neat.

Reply:
Done